src/components/Register/Form.tsx

```diff
@@ -0,0 +1,306 @@
+import React, { useEffect, useState, SyntheticEvent } from 'react';
+import axios from 'axios';
+import * as emailValidator from 'email-validator';
+import {
+  Autocomplete,
+  TextField,
+} from '@mui/material';
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { DesktopDatePicker } from '@mui/x-date-pickers/DesktopDatePicker';
+import { InputContainer, DropDownContainer } from './FormComponents';
+import { StyledButton } from '../../theme';
+import styles from './styles.module.css';
+import Toast from '../Toast';
+import { BACKEND_URL } from '../../../config/config';
+import { FormProps, CollegeObject } from './types';
+import {
+  GenderList,
+  StreamList,
+  YearOfStudyList,
+  interestsList,
+  convertDateFormat,
+} from './formUtils';
+
+export default function RegisterForm({
+  isRegistered,
+  handleRegister,
+}: FormProps) {
+  const [userName, setUserName] = useState<string>('');
+  const [userEmail, setUserEmail] = useState<string>('');
+  const [collegeData, setCollegeData] = useState<CollegeObject[]>([
+    { id: 0, college_name: 'Other' },
```

Comment:
Let this be a checkbox instead. Clicking it should show a text input beside the checkbox and disable the drop down.